src/main/java/seedu/address/model/person/Company.java

```diff
@@ -36,6 +36,9 @@ public Company(String company) {
     public static boolean isValidCompany(String test) {
         return test.matches(VALIDATION_REGEX);
     }
+    public String getCompanyName() {
+        return value;
```

Comment:
[ This comment is for bookkeeping purposes as the original comment was made F2F ]

This shouldn't be needed as all other Person-related fields are accessed via their value fields as well. Company also follows this behaviour